<launch>
    
    <!-- drone_node -->
    <arg name="default_height" default="3.0"/>
    <arg name="timeout_in" default="5.0"/>
    <arg name="timeout_from_commandstart" default="3.0"/>
    <arg name="timeout_connection" default="5.0"/>


    <!-- drone_node -->
    <param name="default_height" value="$(arg default_height)"/>
    <param name="timeout_in" value="$(arg timeout_in)"/>
    <param name="timeout_start" value="$(arg timeout_from_commandstart)"/>
    <param name="timeout_connection" value="$(arg timeout_connection)"/>


    <node name = "drone_node" pkg="tracking" type="drone_node" output="screen"/>
</launch>